src/main/webapp/src/test.ts

```diff
@@ -1,6 +1,14 @@
 // This file is required by karma.conf.js and loads recursively all the .spec and framework files
 
 import 'zone.js/dist/zone-testing';
+
+import 'zone.js/dist/proxy';
```

Comment:
Could you stick in a comment here as to why these imports are needed?

Reply:
I added a comment, but I am, honestly, not 100% what is the exact purpose of these imports...